### ISSUE TITLE
hermione merge-reports fails if images directory doesn’t exists in source report

### ISSUE THREAD

**Issue by isqua** [gemini-testing/html-reporter#142]
**How to reproduce**

1. Write some good tests which doesn’t produce images
2. Run merge-reports

```
npx hermione merge-reports hermione-reports/e2e-one hermione-reports/e2e-two hermione-reports/e2e-three -d hermione-reports/e2e

Error: ENOENT: no such file or directory, link './hermione-reports/e2e-one/images' -> './hermione-reports/e2e/images'
```

**Comment by DudaGod** [comment]
Hello.

Thank you. I will fix it at the weekend.

**Comment by DudaGod** [comment]
Fixed -> html-reporter@2.21.2
